### ISSUE TITLE
run mech -h

### ISSUE THREAD

**Issue by eucariop** [mechboxes/mech#33]
Before installing:
```
   pip install --trusted-host pypi.python.org pip mech
```

Run command line:
```
   mech -h
```

Traceback (most recent call last):
```
  File "C:\Users\EP998234\AppData\Local\Programs\Python\Python36-32\Scripts\mech-script.py", line 11, in <module>
    load_entry_point('mech==0.6', 'console_scripts', 'mech')()
  File "c:\users\ep998234\appdata\local\programs\python\python36-32\lib\site-packages\mech\__main__.py", line 32, in main
    from mech import Mech
ImportError: cannot import name 'Mech'
```

**Comment by ColdHeat** [comment]
Python 3 is currently not supported, you can watch #26

**Comment by Hlsgs** [comment]
Same here, and I'm using Python 2. Tried both 32 and 64-bit.

**Comment by Kronuz** [comment]
This should be fixed in v0.7.0+
